test/integration/mysql-psc/mysql_psc_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql_psc
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMySqlPscModule(t *testing.T) {
+
+	mySql := tft.NewTFBlueprintTest(t)
+
+	mySql.DefineVerify(func(assert *assert.Assertions) {
+		mySql.DefaultVerify(assert)
+
+		instaceNames := []string{mySql.GetStringOutput("name")}
+		op := gcloud.Run(t, fmt.Sprintf("sql instances describe %s --project %s", instaceNames[0], mySql.GetStringOutput("project_id")))
```

Comment:
nit: Let's use runf rather than run+ sprintf

https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test@v0.8.0/pkg/gcloud#Runf